package.json

```diff
@@ -12,7 +12,7 @@
     "client": "npm run start --prefix client",
     "server": "nodemon index.js",
     "dev": "concurrently \"npm run server\" \"npm run client\"",
-    "heroku-postbuild": "NPM_CONFIG_PRODUCTION=false npm install --prefix client && npm run build --prefix client"
+    "heroku-postbuild": "NPM_CONFIG_PRODUCTION=true npm install --prefix client && npm run build --prefix client"
```

Comment:
@kentpmckinney I know you mentioned it before, but I am trying to remember what changing `NPM_CONFIG_PRODUCTION` to true does.